libs/cms/src/lib/cms.module.ts

```diff
@@ -5,7 +5,6 @@ import {
   CmsResolver,
   ArticleResolver,
   LatestNewsSliceResolver,
-  AboutSubPageResolver,
```

Comment:
The AboutSubPageResolver utilizes `AboutPage` which is no longer in the content model. I was aiming to refactor AboutSubPage out of the project in a separate pull request but this deletion has to happen here.